src/test/java/mate/academy/validation/EmailValidatorTest.java

```diff
@@ -0,0 +1,66 @@
+package mate.academy.validation;
+
+import javax.validation.ConstraintValidatorContext;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.mockito.Mockito;
+
+class EmailValidatorTest {
+    private EmailValidator emailValidator;
+    private ConstraintValidatorContext constraintValidatorContext;
+    
+    @BeforeEach
+    void setUp() {
+        constraintValidatorContext = Mockito.mock(ConstraintValidatorContext.class);
+        emailValidator = new EmailValidator();
+    }
+    
+    @Test
+    void isValid_ok() {
+        String email = "testuser@i.ua";
+        boolean actual = emailValidator.isValid(email, constraintValidatorContext);
+        Assertions.assertTrue(actual);
+    }
+    
+    @Test
+    void isValid_NullEmail_notOk() {

```

Comment:
```suggestion
    void isValid_nullEmail_notOk() {
```
the same for methods below